layouts/partials/header.html

```diff
@@ -4,6 +4,13 @@
       {{ .Site.Title }}
     </a>
     {{ if or .Site.Menus.main .Site.IsMultiLingual }}
+    <span id="dark-mode-toggle" class="menu-button float-right">
+      <svg width="24" height="24" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We can use FontAwesome icons instead of a custom SVG.
https://fontawesome.com/icons/adjust?style=solid